src/common/transformations/src/transformations/common_optimizations/leaky_relu_fusion.cpp

```diff
@@ -30,6 +30,17 @@ ov::pass::LeakyReluFusion::LeakyReluFusion() {
         if (shape_size(original_alpha_pattern.get_shape()) != 1)
             return false;
 
+        auto constant = ov::as_type_ptr<op::v0::Constant>(original_alpha_pattern.get_node_shared_ptr());
+        if (!constant)
+            return false;
+
+        float value;
```

Comment:
it can be, but fp64 models are very rare and the constant would have to be at minimum something like 1.0000001 to make it slightly incorrect. Also given that plugins convert f64 to f32 - we won't see the difference anyway.